wallet-web/common/helpers.ts

```diff
@@ -138,11 +137,11 @@ export const basicRawCoinValueValidation = (rawAmount: string): boolean => {
 }
 
 export const getDisplayExecGasFee = (): string => {
-    const table = buildFeeTable(nymGasPrice(UDENOM), nymGasLimits, nymGasLimits)
+    const table = buildFeeTable(nymGasPrice(DENOM), nymGasLimits, nymGasLimits)
```

Comment:
Are you sure `UDENOM` should be changed to `DENOM` here?

Reply:
As discussed this updated the gas fee alert to display PUNK instead of UPUNK. It shouldn't break anything else but happy to revisit once you've given it a test.